argus/action_manager/windows.py

```diff
@@ -239,10 +238,11 @@ def remove(self, path):
     def rmdir(self, path):
         """Remove a directory."""
         if not self.exists(path):
-            raise exceptions.ArgusCLIError("Invalid Path.")
+            raise exceptions.ArgusCLIError("Invalid Path '{}'.".format(path))
 
         if not self.is_file(path):
-            raise exceptions.ArgusCLIError("The path is not a directory.")
```

Comment:
So if it is not a file, the path is not a directory? Correct me if I'm mistaken, but should't it be, `The path {} is not a file` since you are checking if the path is not a file, or am I missing something?